server.js

```diff
@@ -1,14 +1,42 @@
 require("express-async-errors");
 const express = require("express");
 const path = require("path");
+const passport = require('passport');
```

Comment:
Please add a link to the Twitter clubhouse ticket

Reply:
Here it is: https://app.clubhouse.io/startng/story/43811/register-route-implement-twitter-auth

Reply:
Sorry, I'm a bit confused. Did you mean I should add the link within the code? If so, where would the link go in there?